build/root/WORKSPACE

```diff
@@ -1,8 +1,8 @@
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "66282d078c1847c2d876c02c5dabd4fd57cc75eb41a9668a2374352fa73b4587",
-    strip_prefix = "rules_go-ff7e3364d9383cf14155f8c2efc87218d07eb03b",
-    urls = ["https://github.com/bazelbuild/rules_go/archive/ff7e3364d9383cf14155f8c2efc87218d07eb03b.tar.gz"],
+    sha256 = "1cac93a87939915f26844b92df4f6b946f68707a4830a489c3382223e7913fbd",
+    strip_prefix = "rules_go-78c989a6002e1ee28f3a18e3d8d2787feb0fe3a6",
+    urls = ["https://github.com/bazelbuild/rules_go/archive/78c989a6002e1ee28f3a18e3d8d2787feb0fe3a6.tar.gz"],
```

Comment:
maybe we should just switch to the release tarball directly? i.e. https://github.com/bazelbuild/rules_go/releases/download/0.10.2/rules_go-0.10.2.tar.gz.

this won't need the strip_prefix, either.
```
$ sha256sum rules_go-0.10.2.tar.gz 
4b2c61795ac2eefcb28f3eb8e1cb2d8fb3c2eafa0f6712473bc5f93728f38758  rules_go-0.10.2.tar.gz
```